manifests/init.pp

```diff
@@ -27,14 +27,15 @@
   $conf_dir       = '/etc/mesos',
   # e.g. zk://localhost:2181/mesos
   $zookeeper      = '',
+  $quorum         = 1,
   # if "zk" is empty, master value is used
   $master         = '127.0.0.1',
   $master_port    = 5050,
   $owner          = 'root',
   $group          = 'root',
   # could be a fact like $::ipaddress or explicit ip address
-  $listen_address = undef,
-  $repo           = undef,
+  $listen_address = '0.0.0.0',
+  $repo           = 'mesosphere',
```

Comment:
We seem to only support RedHat and Debian and both treat `'mesosphere'` value
https://github.com/deric/puppet-mesos/blob/master/manifests/repo.pp#L20-L21
https://github.com/deric/puppet-mesos/blob/master/manifests/repo.pp#L38-L39

Not sure if this should break anything, but I could just inject it from our config, if you think it's better. 
LMK